pkg/webhookconfig/registration.go

```diff
@@ -538,8 +538,8 @@ func (wrc *Register) constructVerifyMutatingWebhookConfig(caData []byte) *admreg
 				true,
 				wrc.timeoutSeconds,
 				admregapi.Rule{
-					Resources:   []string{"deployments"},
-					APIGroups:   []string{"apps"},
+					Resources:   []string{"Lease"},
```

Comment:
Can we add `objectSelector` to select this particular lease object?

Reply:
Have added the labels would need more guidance on where to use label selector, thanks

Reply:
Fixed this too